bottlerocket-settings-models/settings-extensions/kubernetes/src/lib.rs

```diff
@@ -98,6 +98,9 @@ pub struct KubernetesSettingsV1 {
     seccomp_default: bool,
     #[cfg(feature = "nvidia-device-plugin")]
     device_plugins: K8sDevicePluginsSettings,
+    // Generated in `k8s-1.27+` variants only
+    serialize_image_pulls: bool,
+    max_parallel_image_pulls: i32,
```

Comment:
Are these values not supported on older versions of `kubelet`?

If we add them like this, they'll still be in the settings API for older `*-k8s-*` builds.

Reply:
Correct, this feature landed in k8s-1.27: https://kubernetes.io/docs/concepts/containers/images/#maximum-parallel-image-pulls.

> If we add them like this, they'll still be in the settings API for older *-k8s-* builds.

Also correct - my thought was that these settings would be ignored/unused in k8s < 1.27, and only included in a [`kubelet-config`](https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/develop/packages/kubernetes-1.27/kubelet-config) in kubernetes-1.27+ packages. We'd call out this distinction on the bottlerocket.dev Settings Index

Open to suggestions on a better approach here, though